src/wasm/wasm-debug.cpp

```diff
@@ -473,6 +473,10 @@ struct LocationUpdater {
   // Map start of line tables in the debug_line section to their new locations.
   std::map<BinaryLocation, BinaryLocation> debugLineMap;
 
+  // Map offset of location list entries in the debug_loc section to the base
+  // address of the compilation units referencing them.
+  std::map<uint64_t, BinaryLocation> debugLocMap;
+
```

Comment:
why `int64_t`? In general I think we use `BinaryLocation` for all offsets in the binary, so we could use it here too, unless location lists are special?

Reply:
done